linux_os/guide/system/permissions/files/permissions_var_log_dir/file_permissions_var_log_gdm/rule.yml

```diff
@@ -0,0 +1,23 @@
+documentation_complete: true
+
+title: 'Verify Permissions on /var/log/gdm directory'
+
+description: |-
+    {{{ describe_file_permissions(file="/var/log/gdm", perms="0770") }}}
+
+rationale: |-
+    The <tt>/var/log/gdm</tt> directory contains information about the GDM daemon
+    and should only be accessed by authorized personnel.
+
+severity: medium
+
+ocil_clause: '{{{ ocil_clause_file_permissions(file="/var/log/gdm", perms="-rwxrwx---") }}}'
+
+ocil: |-
+    {{{ ocil_file_permissions(file="/var/log/gdm", perms="-rwxrwx---") }}}
+
+template:
+    name: file_permissions
+    vars:
+        filepath: /var/log/gdm/
+        filemode: '0770'
```

Comment:
I believe this should be 0660

Reply:
gdm is a directory, not a file so the x bit needs to be set.